src/shorthands/JImage.jl

```diff
@@ -0,0 +1,17 @@
+function _JImage(pos, img, centering, shapeargs, shape)
+    if shape != false
+        shape(shapeargs...)
+    end
+    placeimage(img, pos, centered = centering)
```

Comment:
would be nice to be able to scale the image before it gets placed

Reply:
Agreed! Added it in